.appveyor.yml

```diff
@@ -0,0 +1,25 @@
+version: '1.2.0-{build}'
+clone_depth: 1
+test: off
+environment:
+    global:
+        CYG_ROOT: C:\cygwin
+        CYG_MIRROR: http://cygwin.mirror.constant.com
+        CYG_CACHE: C:\cygwin\var\cache\setup
+        CYG_BASH: C:\cygwin\bin\bash
+
+cache:
+    - '%CYG_CACHE%'
+
+init:
+    - git config --global core.autocrlf input
+
+install:
+    - ps: 'Start-FileDownload "http://cygwin.com/setup-x86.exe" -FileName "setup-x86.exe"'
+    - 'setup-x86.exe --quiet-mode --no-shortcuts --only-site --root "%CYG_ROOT%" --site "%CYG_MIRROR%" --local-package-dir "%CYG_CACHE%" --packages gcc-g++,autoconf,automake,make,cmake,libboost-devel,openssl-devel,bzip2,python,git,libstdc++6-devel,libkrb5-devel > NUL 2>&1'
+    - '%CYG_BASH% -lc "cygcheck -dc cygwin"'
+
+build_script:
+    - '%CYG_BASH% -lc "cd $APPVEYOR_BUILD_FOLDER && make gtest tarball"'
+
+# vim: set tw=0:
```

Comment:
This is a [vim modeline](http://vim.wikia.com/wiki/Modeline_magic). I've removed it but it's useful sometimes to have it if you always find yourself setting the same settings on the same file. Now that the line is shorter through your suggestion above, we don't need this anymore.